WordPressAuthenticator/Signin/LoginViewController.swift

```diff
@@ -262,20 +262,33 @@ extension LoginViewController {
         guard let serviceName = loginFields.meta.socialService, let serviceToken = loginFields.meta.socialServiceIDToken else {
             return
         }
-
-        linkSocialService(serviceName: serviceName, serviceToken: serviceToken, wpcomAuthToken: wpcomAuthToken)
+        
+        linkSocialService(serviceName: serviceName,
+                          serviceToken: serviceToken,
+                          wpcomAuthToken: wpcomAuthToken,
+                          appleEmail: loginFields.meta.appleUser?.email,
+                          appleFullName: loginFields.meta.appleUser?.fullName)
     }
 
     /// Links the current WordPress Account to a Social Service.
     ///
-    func linkSocialService(serviceName: SocialServiceName, serviceToken: String, wpcomAuthToken: String) {
-        guard serviceName == .google else {
+    func linkSocialService(serviceName: SocialServiceName,
+                           serviceToken: String,
+                           wpcomAuthToken: String,
+                           appleEmail: String? = nil,
```

Comment:
Same comment as in WPKit, I'm not sure about including the apple parameters directly here. Perhaps we could add them to a parameters dictionary and pass it in?